### PR TITLE
bug fix issue 139, Wrong parameter used as sleep timer cap

### PR DESCRIPTION
**Pull request by sfc-gh-ext-simba-dl**
https://github.com/snowflakedb/snowflake-sdks-drivers-issues-teamwork/issues/139

used incorrect parameter for cap.

### CODE REVIEW

**Comment by sfc-gh-dszmolka** [comment]
#301

File: lib/services/sf.js [hunk 1]
```diff
@@ -1136,7 +1136,8 @@ StateConnecting.prototype.continue = function ()
     Date.now() : 'FIXEDTIMESTAMP';
   const maxLoginRetries = connectionConfig.getRetrySfMaxLoginRetries();
   let sleep = connectionConfig.getRetrySfStartingSleepTime();
-  const cap = connectionConfig.getRetrySfMaxNumRetries();
+  const cap = connectionConfig.getRetrySfMaxSleepTime();
```
Comment (sfc-gh-igarish) [inline lib/services/sf.js:1139]: is it possible sleeptime as 0? Do we have test already exist for sleep time? are we still using max num of retries some where? Make sure "cap" variable meaning change shouldn't cause any side effect.

Comment (sfc-gh-ext-simba-dl) [reply lib/services/sf.js:1139]: we don't have sleeptime = 0. we still use the max num of retries. for example, retry backoff is 1s, 2s, 4s, 8s, 16s, so this is max retry = 5, if max retry = 6, the backoff will looks like 1s, 2s, 4s, 8s, 16s, 16s. Just the max backoff is 16s. And this setting we didn't allow customer modify it.

**Review by sfc-gh-igarish — APPROVED** [review]
LGTM

**Pull request merged**